### PR TITLE
fix: token input decimals

### PR DESCRIPTION
**Pull request by 0xtiti**
closes SUP-93

### CODE REVIEW

File: src/containers/BridgeCard/TokenSection.tsx [hunk 1]
```diff
@@ -122,15 +109,6 @@ const StyledInput = styled(TextField)(() => {
   return {
     padding: '0 0.8rem 0 0',
 
-    '& .MuiInputBase-root::after, & .MuiInputBase-root::before, & .MuiInputBase-root:hover': {
-      content: 'none',
-    },
```
Comment (0xArdy) [inline src/containers/BridgeCard/TokenSection.tsx:127]: lol some styles broke, sorry i think those lines were necessary
image

**Review by 0xArdy — CHANGES_REQUESTED** [review]

File: src/containers/BridgeCard/TokenSection.tsx (hunk 1 above)
Comment (0xtiti) [reply src/containers/BridgeCard/TokenSection.tsx:127]: 897e445bb1aed59abb077fb042233ef8201371b3

**Review by 0xArdy — APPROVED** [review]
lgtm! :rocket:

File: src/containers/BridgeCard/TokenSection.tsx [hunk 2]
```diff
@@ -20,7 +19,8 @@ export const TokenSection = () => {
   const inputValue = isEth ? ethValue : amount;
 
   const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    isEth ? setEthValue(e.target.value) : setAmount(e.target.value);
+    const normalizedInput = e.target.value.replace(/[^0-9.]+/g, '').replace(/(\..*)\..*/g, '$1');
+    isEth ? setEthValue(normalizedInput) : setAmount(normalizedInput);
```
Comment (turtlemoji) [inline src/containers/BridgeCard/TokenSection.tsx:23]:
```suggestion
    if (isEth) {
         setEthValue(normalizedInput)
    } else {
        setAmount(normalizedInput);
    }
```

File: src/containers/BridgeCard/TokenSection.tsx [hunk 3]
```diff
@@ -20,7 +19,8 @@ export const TokenSection = () => {
   const inputValue = isEth ? ethValue : amount;
 
   const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    isEth ? setEthValue(e.target.value) : setAmount(e.target.value);
+    const normalizedInput = e.target.value.replace(/[^0-9.]+/g, '').replace(/(\..*)\..*/g, '$1');
```
Comment (turtlemoji) [inline src/containers/BridgeCard/TokenSection.tsx:22]: A comment could add value here, so, it seems to remove everything except numbers and dots right?

File: src/containers/BridgeCard/TokenSection.tsx (hunk 2 above)
Comment (turtlemoji) [reply src/containers/BridgeCard/TokenSection.tsx:23]: just for readability

**Review by turtlemoji — APPROVED** [review]
pre-approved with some comments

File: src/containers/BridgeCard/TokenSection.tsx (hunk 3 above)
Comment (0xtiti) [reply src/containers/BridgeCard/TokenSection.tsx:23]: https://github.com/defi-wonderland/superchain-bridge-ui/pull/57/commits/5c86f870db2191d10e3924cf3605277363aabaec

Comment (0xtiti) [reply src/containers/BridgeCard/TokenSection.tsx:23]: https://github.com/defi-wonderland/superchain-bridge-ui/pull/57/commits/5c86f870db2191d10e3924cf3605277363aabaec

File: src/containers/BridgeCard/TokenSection.tsx (hunk 2 above)
Comment (0xtiti) [reply src/containers/BridgeCard/TokenSection.tsx]: https://github.com/defi-wonderland/superchain-bridge-ui/pull/57/commits/5c86f870db2191d10e3924cf3605277363aabaec

**Review by 0xArdy — APPROVED** [review]

**Pull request merged**